File: sbe/binance/spot_3_2/generated-python/spot_sbe/RateLimitInterval.py
"""Generated SBE (Simple Binary Encoding) message codec."""

import struct
from typing import List, Optional, Tuple
from io import BytesIO

class RateLimitInterval:
    """Enum values."""
    SECOND = 0
    MINUTE = 1
    HOUR = 2
    DAY = 3
    NON_REPRESENTABLE = 254
    NULL_VALUE = 255

    @staticmethod
    def encoded_length() -> int:
        return 1

File: sbe/binance/spot_3_2/generated-php/spot_sbe/RateLimitInterval.php
<?php
/**
 * Generated SBE (Simple Binary Encoding) message codec.
 */

class RateLimitInterval
{
    public const SECOND = 0;
    public const MINUTE = 1;
    public const HOUR = 2;
    public const DAY = 3;
    public const NONREPRESENTABLE = 254;
    public const NULL_VALUE = 255;

    public static function encodedLength(): int
    {
        return 1;
    }
}
